src/components/Settings/SettingsAbout/index.tsx

```diff
@@ -86,7 +94,7 @@ const SettingsAbout: React.FC = () => {
               rel="noreferrer"
               className="text-indigo-500 hover:underline"
             >
-              {intl.formatMessage(messages.clickheretojoindiscord)}
+              https://discord.gg/PkCWJSeCk7
```

Comment:
What was wrong with the descriptive string?

Reply:
It doesn't match the pattern on that page:
![image](https://user-images.githubusercontent.com/52870424/111487377-af94f980-870e-11eb-9fd6-1847402aa696.png)

Reply:
There is no set-in-stone pattern for this page, though. I don't like the invite URL being just the plain invite code because technically, it can change. Unless we get level 3 boosting going on and have a static URL, I like the other string better.

Reply:
I suppose that could happen? Either way, it's not important. No need to bikeshed over the specific text of a link. If you like the URL more, we can go with that.